amgut/test/test_dbpatch_source.py

```diff
@@ -0,0 +1,104 @@
+from unittest import TestCase, main
```

Comment:
It becomes an issue if you run nosetests on your machine twice without resetting the DB manually.